Docs/Configuration.tex

```diff
@@ -1592,52 +1576,51 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   This option performs memory map analysis of the firmware and checks whether
   all slides (from \texttt{1} to \texttt{255}) can be used. As \texttt{boot.efi}
   generates this value randomly with \texttt{rdrand} or pseudo randomly \texttt{rdtsc},
-  there is a chance of boot failure when it chooses a conflicting slide. In case
-  potential conflicts exist, this option forces macOS to use a pseudo random value
-  among the available ones. This also ensures that \texttt{slide=} argument is never
-  passed to the operating system for security reasons.
+  there is a chance of boot failure when it chooses a conflicting slide. In cases where
+  potential conflicts exist, this option forces macOS to select a pseudo random value
+  from the available values. This also ensures that the \texttt{slide=} argument is
+  never passed to the operating system for security reasons.
```

Comment:
suggest add comma: "never passed to the operating system, for security reasons" (there are two possible meanings here, and the one more clearly implied with the comma - i.e. "it is _never_ passed, and this is why" (as opposed to: "it is never passed for this reason, but is sometimes passed (for other reasons)") - I assume is correct!)

Reply:
Used brackets to make clearer